fpga/pulpissimo-zedboard/ips/xilinx_clk_mngr/tcl/run.tcl

```diff
@@ -1,14 +1,4 @@
-set partNumber $::env(XILINX_PART)
-
-if [info exists ::env(BOARD)] {
-    set BOARD $::env(BOARD)
-} else {
-    error "BOARD is not defined. Please source the sourceme.sh file."
-    exit
-}
-if [info exists ::env(XILINX_BOARD)] {
-	set boardName  $::env(XILINX_BOARD)
-}
+source ../../tcl/common.tcl
 
```

Comment:
I guess this make sense to hardcode it.

Reply:
I am not sure if that's just a comment because I am not sure what you mean exactly by "hardcode" and "it". Please let me know what/if I should change something.

Thanks for looking into it BTW.